main.go

```diff
@@ -39,15 +42,18 @@ import (
 var (
 	scheme   = runtime.NewScheme()
 	setupLog = ctrl.Log.WithName("setup")
-
-	version string // value injected in compilation-time
+	logLevel = common.FetchEnv("LOG_LEVEL", "info")
+	logMode  = common.FetchEnv("LOG_MODE", "production")
```

Comment:
We've been moving away from env vars to configure the components, to favour command-line args instead.

Authorino still supports some env vars, including ones to configure the logging system, but that's only for backward compatibility. In all the cases, command-line args take precedence.

- https://github.com/Kuadrant/authorino/blob/8addd6752d9d424d295a70f6d75835069fdce0a6/main.go#L213-L214

In the Operator, I think even more so we should avoid introducing new env vars (because currently we have none.) Would you mind switching to command-line args instead please?